plugins/polio/js/src/constants/virus.ts

```diff
@@ -55,4 +58,36 @@ const polioVaccines: PolioVaccine[] = [
     },
 ];
 
-export { polioVaccines, polioViruses };
+export const useOtherVaccineColor = (): string => {
+    const theme = useTheme();
+    return theme.palette.secondary.main;
+};
```

Comment:
Not sure I would do that. I'd rather copy paste the hex value in another constant. There's no reason to tie the map color to the theme